Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2184,24 +2183,26 @@ InstanceInfoRecord UpdateInstanceInfo(InstanceInfoData data)
                     }
 
                     SetInstanceTransform(instancedCopy, instanceRendererParent, mat);
-
-                    if (config.SyncVisibility &&
-                        instancedEntityRecord.hasVisibility) {
-                        instancedCopy.SetActive(visible);
-                    }
-
+                    EnableInstancedCopy(instancedCopy);
```

Comment:
Shouldn't we keep this check ?
```
                    if (config.SyncVisibility &&
                        instancedEntityRecord.hasVisibility
```

Reply:
I think we should remove it. The reason is that even when we sync visibility, we should still enable the copies for the following reasons:

In Blender, only the parent object visibility controls the visibility of the instances in the scene.

In Unity, since we create copies as children of the parent, the copies inherit the visibility status of the parent by default.

So the only thing we need to make sure is synched when it comes to visibility, is the parent visibility.
Before, we used to sync the visibility of each copy with the visibility of the referenced object in the blender scene. If the referenced object was invisible, it would lead to this result
![copyInstancedActiveIssue](https://user-images.githubusercontent.com/51912249/207615871-8bc4fc3f-1982-4f54-a096-f0878f249f7a.png)

where instances render in blender but not in Unity.

So for visibility to sync correctly, we need to always enable the copies so they will only be disabled only if the parent was disabled.